packages/lesmis/src/main.jsx

```diff
@@ -22,10 +22,11 @@ const render = () => {
 
 render(App);
 
-if (module.hot) {
-  module.hot.accept('./App', () => {
+if (import.meta.hot) {
+  import.meta.hot.accept('./App', () => {
     // eslint-disable-next-line global-require
     const NextApp = require('./App').default;
     render(NextApp);
   });
 }
+ 
```

Comment:
```suggestion

```
Actually can we just remove this code. It is just a work around for the fact that react-scripts hot reload didn't work properly.